src/components/Chat/ChatMessage.tsx

```diff
@@ -120,9 +120,20 @@ class ChatMessage extends UIComponent<ReactProps<ChatMessageProps>, ChatMessageS
         ) : (
           <>
             {!mine &&
-              Text.create(author, { defaultProps: { size: 'small', styles: styles.author } })}
+              Text.create(author, {
+                defaultProps: {
+                  size: 'small',
+                  styles: styles.author,
+                  className: `${ChatMessage.className}__author`,
```

Comment:
This was needed for targeting this className in the chat item for displaying or not the author of the message. If we add this prop on the ChatMessage as well, we won't need this.